FinniversKit/Sources/SwiftUI Components/Button/SwiftUIFloatingButton.swift

```diff
@@ -21,8 +21,10 @@ struct FloatingButtonStyle: ButtonStyle {
     }
 }
 
-struct SwiftUIFloatingButton_Previews: PreviewProvider {
-    static var previews: some View {
+public struct SwiftUIFloatingButton_Previews: PreviewProvider {
```

Comment:
Any other way around this? Public previews feels off. 🤔

Reply:
There hopefully is, but I just wanted the script to work now. 

I guess either replicate those in the demo project makes the most sense, either that or we will still have demo/preview things available publicly in FinniversKit 🤔

Reply:
I copied them instead of making public, it is better that way I think, do you mind having another look? 😄